docs/api/symbol-package-publish-resource.md

```diff
@@ -0,0 +1,90 @@
+
+## Versioning
+
+The following `@type` value is used:
+
+@type value          | Notes
+-------------------- | -----
+SymbolPackagePublish/4.9.0 | The initial release
+
+## Base URL
+
+The base URL for the following APIs is the value of the `@id` property of the `SymbolPackagePublish/4.9.0` resource in the
+package source's [service index](service-index.md). For the documentation below, nuget.org's URL is used. Consider 
+`https://www.nuget.org/api/v2/symbolpackage` as a placeholder for the `@id` value found in the service index.
+
+
+## HTTP methods
+
+The `PUT` HTTP method is supported by this resource. 
+
+## Push a symbol package
+
+> [!Note]
+> nuget.org has [additional requirements](NuGet-Protocols.md) for interacting with the push endpoint. For snupkg constraints see [symbol package constraints](../create-packages/Symbol-Packages-V2.md).
+
+nuget.org supports pushing new symbol packages format ([snupkg](../create-packages/Symbol-Packages-V2.md)) using the following API. Symbol package with the provided ID and version
+can be submitted multiple times. A symbol package will be rejected in the following cases.
+- A package with the same Id and Version does not exist.
+- A symbol package with the same Id and Version was pushed but is not yet published.
+- The symbol package ([snupkg](../create-packages/Symbol-Packages-V2.md)) is invalid (see [symbol package constraints](../create-packages/Symbol-Packages-V2.md)).
+
+Other package sources may support different contracts.
+
+    PUT https://www.nuget.org/api/v2/symbolpackage
+
+### Request parameters
+
+Name           | In     | Type   | Required | Notes
+-------------- | ------ | ------ | -------- | -----
+X-NuGet-ApiKey | Header | string | yes      | For example, `X-NuGet-ApiKey: {USER_API_KEY}`
+
+The API key is an opaque string gotten from the package source by the user and configured into the client. No
+particular string format is mandated but the length of the API key should not exceed a reasonable size for HTTP header
+values.
+
+### Request body
+
+The request body must come in the following form:
+
+#### Multipart form data
+
+The request header `Content-Type` is `multipart/form-data` and the first item in the request body is the raw bytes of
+the .snupkg being pushed. Subsequent items in the multipart body are ignored. The file name or any other headers of the
+multipart items are ignored.
+
+### Response
+
+Status Code | Meaning
+----------- | -------
+201, 202    | The symbol package was successfully pushed.
+400         | The provided symbol package is invalid.
+401         | The user is not authorized to perform this action.
+404         | A coresponding package with the provided ID and version does not exist.
+409         | A symbol package with the provided ID and version was pushed but it is not available yet.
+
+Server implementations vary on the success status code returned when a package is successfully pushed.
```

Comment:
This statement should not be true. We should assert the single status code the gallery returns today.

Reply:
Updated.